tests/resources/sample_lyrics.py

```diff
@@ -0,0 +1,81 @@
+KO_MANGGASONG = """
```

Comment:
.py file in test/resources folder directly is bad idea. (resources only for non-py files like .txt or .md)

Move variables directly to `test_preprocess.py` file is better.